src/bowlingScorer.js

```diff
@@ -0,0 +1,68 @@
+var BowlingScorer = function() {
+  this.totalScore = 0
+  this.frameScore = [,]
+  this.frameTotal = 0
+  this.frame = 1
+  this.roll = 1
+  this.scoreChart = [ , , , , , , , , , ]
+};
+
+BowlingScorer.prototype.returnScore = function() {
+  return this.totalScore
+};
+
+BowlingScorer.prototype.returnFrame = function() {
+  return this.frame
+};
+
+BowlingScorer.prototype.returnFrameScore = function(frame) {
+  return this.scoreChart[frame-1]
+};
+
+BowlingScorer.prototype.returnBowlScore = function(score) {
```

Comment:
Not sure what the point of this function is? Why return the value that you pass in?